<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>20</width>
      <height>331</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_xem">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>2.Xem chi tiết khấu hao</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
      <pointsize>14</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 57 14pt &quot;Poppins Medium&quot;;

</string>
    </property>
    <property name="text">
     <string>Tính Khấu Hao TSCĐ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_giamuamoi">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Giá mua mới:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_giamuamoi">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_phivanchuyen">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Phí vận chuyển:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_phivanchuyen">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_philapdat">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Phí lặp đặt:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_philapdat">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_sonamsd">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Số năm sử dụng dự kiến:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sonamsd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tinh">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>1.Tính khấu hao</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nguyengia">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nguyên giá TSCĐ:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nguyengia">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_khauhaonam">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mức trích khấu hao năm:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_khauhaotham">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mức trích khấu hao tháng:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_khauhaonam">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_khauhaothang">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>301</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
